app/views.py

```diff
@@ -69,13 +82,62 @@ def login():
     else:
         return render_template('login.html')
 
-@app.route('/create/')
-def create():
-    return render_template("create.html")
+@app.route('/create/', methods=['GET', 'POST'])
+def createrecipecats():
+    """Handles creation of recipe categories"""
+    if  'email' in session:
+        if request.method == "POST":
+            sentence = request.form['category']
+            Catlist = sentence.split(' ')
+            category = ''.join(Catlist)
+            owner = session['email']
+            result = newRecipeCat.create(category, owner)
+            print(category)
+            if result == 2:
```

Comment:
What are these values for? `result == 2` You switched to using return messages in some files and left out others